docs/compile-xtrabackup.md

```diff
@@ -109,6 +106,12 @@ $ cmake -DWITH_BOOST=PATH-TO-BOOST-LIBRARY -DDOWNLOAD_BOOST=ON \
 -DBUILD_CONFIG=xtrabackup_release -DWITH_MAN_PAGES=OFF -B ..
 ```
 
+If you use [Pro builds](./pxb-pro.md), run the following command
+
+```{.bash data-prompt="$"}
+$ cmake -DPROBUILD=1
```

Comment:
I think it's this option in addition to the other options?

Reply:
updated